plat/common/include/uk/plat/common/common.lds.h

```diff
@@ -137,7 +137,7 @@
 	}
 
 #define TLS_SECTIONS							\
-	. = ALIGN(0x20);						\
+	. = ALIGN(0x1000);						\
```

Comment:
What about using `__PAGE_SIZE`?
```suggestion
	. = ALIGN(__PAGE_SIZE);						\
```